modules/openapi-generator-cli/src/main/java/org/openapitools/codegen/cmd/Generate.java

```diff
@@ -246,6 +247,11 @@ public void run() {
         }
 
         if (isNotEmpty(spec)) {
+            if (!spec.matches("^http(s)?://.*") && !new File(spec).exists()) {
+                System.out.println("[error] The spec file is not found: " + spec);
```

Comment:
Do you mean to write an error message to System.out here? This would be surprising to me as a user. Seems like maybe a typo?